File: Api-Buen-Sabor/src/main/java/com/example/buensabor/entity/Pedido.java
package com.example.buensabor.entity;

import java.util.Date;

public class Pedido {

    private long id;
    private int numero;
    private Date fecha;
    private int estado;
    private Date horaEstimadaFin;
    private int tipoEnvio;
    private double total;
    private Domicilio domicilio;
    private Factura factura;
    private MercadoPagoDatos mercadoPagoDatos;

}

File: buen-sabor-app/src/app/models/pedido.ts
import { Domicilio } from "./domicilio";
import { PedidoEstado } from "./pedido-estado";
import { TipoEnvio } from "./tipo-envio";
import { Usuario } from "./usuario";

export interface Pedido {
    numeroPedido: number;
    horaEstimadaFin: Date;
    total: number;
    usuario: Usuario;
    tipoEnvio: TipoEnvio;
    pedidoEstado: PedidoEstado;
    domicilio: Domicilio;
    formaPago: string;
}
